layers/+email/mu4e/packages.el

```diff
@@ -58,7 +58,27 @@
 
       (when mu4e-account-alist
         (add-hook 'mu4e-compose-pre-hook 'mu4e/set-account)
-        (add-hook 'message-sent-hook 'mu4e/mail-account-reset)))))
+        (add-hook 'message-sent-hook 'mu4e/mail-account-reset))))
+
+  ;; Source https://github.com/malb/emacs.d/blob/48508219453ebb542e6fec370bb49fba9980c2a2/malb.org
+  (add-hook 'dired-mode-hook 'turn-on-gnus-dired-mode)
+  (use-package gnus-dired
```

Comment:
Shouldn't it declared in `mu4e-packages` list and then configured in `mu4e/init-gnus-dired`? Am I missing something? 😸